packages/default-field-editors/src/FieldWrapper.tsx

```diff
@@ -0,0 +1,71 @@
+import * as React from 'react';
+import { cx } from 'emotion';
+import { HelpText, FieldGroup, FormLabel } from '@contentful/forma-36-react-components';
+import { ValidationErrors } from '@contentful/field-editor-validation-errors';
+import type { FieldExtensionSDK, Entry } from '@contentful/field-editor-shared';
+import { styles } from './FieldWrapper.styles';
+
+type FieldWrapperProps = {
+  name: string;
+  sdk: FieldExtensionSDK;
+  getEntryURL: (entry: Entry) => string;
+  className?: string;
+  children: React.ReactNode;
+  renderHeading?: (name: string) => JSX.Element | null;
+  renderHelpText?: (helpText: string) => JSX.Element | null;
```

Comment:
Maybe a bool prop to control whether or not to render the focus bar would make sense, [also see this comment](https://github.com/contentful/creator-app-mvp/pull/536/files#r474497472)